components/calendar/calendar.js

```diff
@@ -213,15 +214,19 @@ export default class Calendar extends Component {
   }
 
   render() {
-    const { dataAttrs = {}, isDaySelectableFn, locale, navButtons, selectionType } = this.props;
+    const { dataAttrs = {}, isDaySelectableFn, locale, navButtons, selectionType, className } = this.props;
```

Comment:
default value for `dataAttrs` is not necessary, because we check it in `getDataAttributes` function

Reply:
Fixed